### PR TITLE
feat(experiments): add Funnel experiments credible intervals UI

### PR DESCRIPTION
**Pull request by jurajmajerik**
## Changes
- Add credible intervals to the results summary
- Add conversion rate % delta (requested by multiple users)
- Small cleanup of a Kea selector

image

## How did you test this code?
👀

### CODE REVIEW

File: frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx [hunk 1]
```diff
@@ -88,13 +88,68 @@ export function SummaryTable(): JSX.Element {
...
+                        <Tooltip title="Delta % indicates the percentage change in the conversion rate between the control and the test variant.">
+                            <IconInfo className="text-muted-alt text-base" />
+                        </Tooltip>
+                    </div>
+                ),
+                render: function Key(_, item): JSX.Element {
+                    if (item.key === 'control') {
+                        return <></>
+                    }
+
```
Comment (dmarticus) [inline frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx:112]: I know why we're returning nothing here (the control is the baseline), but IMO it feels a bit odd to have a table entry with _nothing_ in it, at first glance it makes me feel like there's a bug in the table.  Would it be possible to put a different visual indicator denoting that this cell represents the baseline?

File: frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx [hunk 2]
```diff
@@ -88,13 +88,68 @@ export function SummaryTable(): JSX.Element {
...
+                        item.key
+                    ]
+                    if (!credibleInterval) {
+                        return <>--</>
+                    }
+
+                    const lowerBound = (credibleInterval[0] * 100).toFixed(2)
+                    const upperBound = (credibleInterval[1] * 100).toFixed(2)
+
+                    return <div className="font-semibold">{`[${lowerBound}%, ${upperBound}%]`}</div>
```
Comment (dmarticus) [inline frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx:150]: this is good, I think using closed interval notation is the correct choice.

**Review by dmarticus — COMMENTED** [review]
Approved with 2 pretty specific comments, neither of which you need to acknowledge, but that I felt were worth bringing up anyway.

Nice work!

**Review by dmarticus — APPROVED** [review]

File: frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx [hunk 3]
```diff
@@ -88,13 +88,68 @@ export function SummaryTable(): JSX.Element {
...
+                        >{`${delta > 0 ? '+' : ''}${delta.toFixed(2)}%`}</div>
+                    )
+                },
+            }),
+            columns.push({
+                key: 'credibleInterval',
+                title: (
+                    <div className="inline-flex items-center space-x-1">
+                        <div className="">Credible interval</div>
+                        <Tooltip title="A credible interval represents the range within which the true parameter value lies with 95% probability, given the observed data.">
```
Comment (dmarticus) [inline frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx:134]:
```suggestion
                        <Tooltip title="A credible interval represents a range within which we believe the true parameter value lies with a certain probability (often 95%), based on the posterior distribution derived from the observed data and our prior beliefs.">
```

I'm being pedantic here, but maybe consider being a touch more explicit in this tooltip.  My proposal incorporates the following elements:

- emphasizes that the interval is based on the posterior distribution
- mentions that prior beliefs are also incorporated
- avoids implying that 95% is the only possible probability level

Feel free to ignore, but just figured I'd bring it up.

File: frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx (hunk 1 above)
Comment (jurajmajerik) [reply frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx:112]: Good point, opted for this:

image

(the dash empty state I want to only use in cases when there is no result now but may be later)

File: frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx (hunk 3 above)
Comment (jurajmajerik) [reply frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx]: nice, thanks! I think we still need to explain prior/posterior in the docs (issue: https://github.com/PostHog/posthog/issues/24277), but I'll use this for now :)

**Pull request merged**